### ISSUE TITLE
osx-arm64 version of qutip?

### ISSUE THREAD

**Issue by dooleysh** [conda-forge/qutip-feedstock#73]
Hi. I've been trying to install qutip on a new Mac with the arm64 processor (with the command `conda install -c conda-forge qutip`). However, it seems that qutip is not available on any of my conda channels. Do you know if there are any plans to make qutip available on an osx-arm64 channel? Thanks.

My error message:

```
conda install -c conda-forge qutip
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - qutip

Current channels:

  - https://conda.anaconda.org/conda-forge/osx-arm64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/osx-arm64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/osx-arm64
  - https://repo.anaconda.com/pkgs/r/noarch

```

**Comment by jakelishman** [comment]
I think both `conda` and `cibuildwheel` (how we build for `pip`) both support building on M1 Mac now, so we probably ought to be able to do this.  Would you mind opening an issue on [qutip/qutip](https://github.com/qutip/qutip) as well?  It'll get a few more eyes that way (I don't have as much time to sort out the build process for qutip any more...).

**Comment by nathanshammah** [comment]
Ping for @hodgestar and whoever may kindly help keep an eye on this.

**Comment by hodgestar** [comment]
I'll try address then when we do the rebuild for Python 3.10, but that has to wait for https://github.com/qutip/qutip/pull/1735 or similar to be merged and for QuTiP 4.6.3 to be released (hopefully in the next week or so).

**Comment by hodgestar** [comment]
QuTiP 4.6.3 has been released, so now we are just waiting for conda-forge builds to work again so that we can finish merging #74. We should check whether that builds for the M1.

**Comment by hodgestar** [comment]
#74 didn't automatically add builds for the M1, but I will try get the robots to open a PR for the M1 builds shortly..

**Comment by hodgestar** [comment]
Robot migration activation PR -- https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2509

**Comment by hodgestar** [comment]
I've merged #76 so hopefully M1 builds should appear in a couple of hours and we can try them out!

**Comment by hodgestar** [comment]
It looks like the OS X ARM64 builds have succeeded. I'm going to close this issue for now. Please give them a try and let me know how it goes.
